src/dlls/dbgshim/dbgshim.cpp

```diff
@@ -1770,6 +1769,8 @@ CreateDebuggingInterfaceFromVersion2(
         goto Exit;
     }
 
+    FPCoreCLRCreateCordbObject fpCreate2 = NULL;
```

Comment:
What was the cppcheck complaining about w.r.t. this change?

Reply:
```
[src/dlls/dbgshim/dbgshim.cpp:1767] -> [src/dlls/dbgshim/dbgshim.cpp:1763]: (warning) Either the condition 'ppCordb==NULL' is redundant or there is possible null pointer dereference: ppCordb.
```

Reply:
It doesn't seem to make sense that moving the fpCreate2 definition location would fix a problem like that. Btw, it seems the fpCreate2 is completely unused, can you please delete it instead?

Reply:
sure

Reply:
done